packages/mobile/__tests__/pages/Login.test.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { render, fireEvent, wait, waitForElement } from '@testing-library/react-native';
+import configureStore from 'redux-mock-store';
+import AsyncStorage from '@react-native-community/async-storage';
+import { showMessage } from 'react-native-flash-message';
+
+import Wrapper from '../../helpers/Wrapper';
+import { handleLoginFacebook } from '../../src/services/authFacebook';
+import Login, { USER_LOGIN_MUTATION } from '~/pages/Login/index';
+import { createApolloStore } from '../../fixtures/login';
+
+const navigation = {
+  dispatch: jest.fn(),
+  navigate: jest.fn(),
+};
+jest.mock('../../src/services/authFacebook');
+jest.mock('@react-native-community/async-storage');
+jest.mock('react-native-flash-message');
+
+describe('Login', () => {
+  const mockStore = configureStore();
+  const store = mockStore({});
+  const apolloStore = createApolloStore(USER_LOGIN_MUTATION);
+
+  const component = (
+    <Wrapper apolloStore={apolloStore} reduxStore={store}>
+      <Login navigation={navigation} />
+    </Wrapper>
+  );
```

Comment:
Maybe you extract this to a method like:

```js
const renderComponnent = () => {
  const mockStore = configureStore();
  const store = mockStore({});
  const apolloStore = createApolloStore(USER_LOGIN_MUTATION);
  const component = (
    <Wrapper apolloStore={apolloStore} reduxStore={store}>
      <Login nagivstion={navigation} />
    </Wrapper>
  );

  return render(component)
}

describe('Login', () => {
...
```

This way, you can call render component like this:
```js
it('...', () => {
  const { getByText } = renderComponnent();
  ...
```
and you will get a new component and store every time and avoid one test impact in another (in some cases).

Reply:
It looks good! Thanks!